### PR TITLE
Centrality classes

### PR DESCRIPTION
**Pull request by Hendrik1704**
This PR implements a CentralityClasses class, where the input is a list or array of multiplicity from events in a certain phase space. Then it generates centrality bins and there is a function to print these to a file.
If you then have a certain dNchdeta and want to know the class, then there is a function that returns the index of the corresponding class as an int.

### CODE REVIEW

File: src/sparkx/CentralityClasses.py [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+    ------
+    TypeError
+        If `events_multiplicity` or `centrality_bins` is not a list or numpy.ndarray.
+
+    Attributes
+    ----------
+    events_multiplicity_ : list or numpy.ndarray
+        Stores the input multiplicity values for each event.
+    centrality_bins_ : list or numpy.ndarray
+        Stores the input boundaries of centrality classes.
```
Comment (NGoetz) [inline src/sparkx/CentralityClasses.py:26]: Why is it called "deta"?

**Review by NGoetz — COMMENTED** [review]
I have just two little questions before we can merge the PR.

File: src/sparkx/CentralityClasses.py [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+            MinRecord = MaxRecord
+
+        # sort events by multiplicity and determine boundaries of centrality classes
+        global_event_record = sorted(self.events_multiplicity_, reverse=True)
+
+        MinRecord = int(number_events * self.centrality_bins_[0] / 100.0)
+        for i in range(1, len(self.centrality_bins_)):
+            MaxRecord = int(number_events * self.centrality_bins_[i] / 100.0)
+
+            self.dNchdetaMax_.append(global_event_record[MinRecord])
```
[the remark this answers is not in the document]
Comment (Hendrik1704) [reply src/sparkx/CentralityClasses.py:150]: Historic reasons...

[the remark this answers is not in the document]
Comment (NGoetz) [reply src/sparkx/CentralityClasses.py:150]: Okay it should not affect the performance too much so we can leave it.

**Review by NGoetz — APPROVED** [review]
Thank you for the clearifications!

**Pull request merged**